src/main/java/org/broadinstitute/hellbender/tools/copynumber/coverage/readcount/ReadCountFileHeaderKey.java

```diff
@@ -0,0 +1,59 @@
+package org.broadinstitute.hellbender.tools.copynumber.coverage.readcount;
```

Comment:
Note that this class came from another branch, and is needed because IntegerReadCountFileComparator is dependent on it. It will be used in my future PR